specs/commandline/verify.md

```diff
@@ -168,3 +170,25 @@ An example of output messages for a successful verification:
 Warning:  Always verify the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
 Successfully verified signature for localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
 ```
+
+### [Preview] Verify local OCI image
+
+Users should configure trust policy properly before verifying local OCI image. According to trust policy specification, `registryScopes` property determines which trust policy is applicable for the given artifact. The value of `registryScopes` should be a fully qualified URL for both local and remote images. For example, a remote image is referenced by "localhost:5000/net-monitor:v1", thus the value of `registryScopes` should contain "localhost:5000/net-monitor", which is the repository URL of the image. Local OCI image is in the form of OCI layout directory, the reference is different from a remote image, for example "./hello-world:v1". Besides flag `--local-artifact`, users need to specify which `registryScopes` is used to verify the local image using flag `--scope`. Here is an example of trust policy configured for local image `./hello-world:v1`:
```

Comment:
`--scope` specifies the reference `<registry>/<repository>` that the artifact is intended to live. It can be viewed that `--scope` overrides the intended scope for any artifact. 

If the artifact is in a remote registry, the scope can be extracted from the reference, and can be override by `--scope`.
For example,
```
notation verify --scope foo.io/net-monitor bar.io/net-monitor@sha256:xxxx
```
The above command tries to verify `bar.io/net-monitor@sha256:xxxx` as if it is in `foo.io/net-monitor`. It is useful for mirrors.

Following the same logic,
```
notation verify --scope foo.io/net-monitor --oci-layout net-monitor.tar@sha256:xxxx
```
The above command tries to verify `sha256:xxxx` in `net-monitor.tar` as if it is in `foo.io/net-monitor`. Since we cannot extract scope from `net-monitor.tar`, the flag `--scope` becomes a required flag.

Reply:
I and Samir had a discussion and agree with what Shiwei has proposed.